<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="QTLSYS1-182" name="tests.QCStore" tests="9" failures="1" timestamp="04 May 2019 18:21:51 GMT" time="3126.518" errors="0">
  <testcase name="JQC_TLP_Deposit_MidDay_Txn" time="301.033" classname="tests.QCStore"/>
  <testcase name="JQC_TLP_GraceDays_Deposit_MidDay_Txn" time="390.026" classname="tests.QCStore"/>
  <testcase name="JQC_TLP_GraceDays_Deposit_Menu_Txn" time="378.353" classname="tests.QCStore"/>
  <testcase name="JQC_TLP_Deposit_dropdown_Txn" time="290.997" classname="tests.QCStore"/>
  <testcase name="JQC_TLP_Deposit_Menu_Txn" time="300.961" classname="tests.QCStore"/>
  <testcase name="JQC_TLP_GraceDays_Deposit_dropdown_Txn" time="374.175" classname="tests.QCStore"/>
  <testcase name="JQC_TLP_Deceased_Txn" time="275.490" classname="tests.QCStore"/>
  <testcase name="JQC_DefaultPayment_Txn_Testdata" time="388.199" classname="tests.QCStore"/>
  <testcase name="JQC_DefaultPayment_Void_Txn_Testdata" time="427.284" classname="tests.QCStore">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at tests.JQCCSRNewLoan.newLoan(JQCCSRNewLoan.java:268)
at tests.QCStore.JQC_DefaultPayment_Void_Txn_Testdata(QCStore.java:1270)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- JQC_DefaultPayment_Void_Txn_Testdata -->
</testsuite> <!-- tests.QCStore -->
